src/OSPSuite.UI/Services/DataImporter.cs

```diff
@@ -109,7 +109,7 @@ DataImporterSettings dataImporterSettings
             }
             catch (Exception e) when (e is UnsupportedFormatException || e is UnsupportedFileTypeException)
             {
-               _dialogCreator.MessageBoxError(e.Message);
+               _dialogCreator.MessageBoxError(e.Message, true);
```

Comment:
This is a very leaky abstracvtion. Here you need to know know that the message as hyperlink. Why do we need a parameter to specify use HTML or not? Cannot we always use it? In my opinion, this looks like some internal stuff. that should not be visible to the caller

Reply:
done